crates/bounded_planet_core/src/land/mod.rs

```diff
@@ -1,5 +1,9 @@
-mod heightmap;
+pub mod heightmap;
 pub use heightmap::TextureHeightmap;
 
-mod mesh;
+pub mod mesh;
+pub use mesh::texture_to_mesh_data;
 pub use mesh::texture_to_mesh;
+pub use mesh::MeshData;
```

Comment:
```suggestion
pub use mesh::{MeshData, texture_to_mesh_data, texture_to_mesh};
```

Reply:
texture_to_mesh_data and texture_to_mesh where alrady seperate use's so I kept that
I will merge them anyway, because they probably should be